### PR TITLE
Add AlphaMixL<>

### PR DESCRIPTION
**Pull request by fdarosa2663**

### CODE REVIEW

File: styles/alpha.h [hunk 1]
```diff
@@ -38,4 +38,6 @@ class AlphaL {
   }
 };
 
+template<class MIX, class... COLORS> using AlphaMixL = AlphaL<Mix<MIX, COLORS...>, MIX>;
```
Comment (profezzorn) [inline styles/alpha.h:41]: Needs comment.

Comment (fdarosa2663) [reply styles/alpha.h:42]: added

File: styles/alpha.h [hunk 2]
```diff
@@ -38,4 +38,7 @@ class AlphaL {
   }
 };
 
+// To enable Gradient/Mixes constricted within Bump<> and SmoothStep<> layers
```
Comment (profezzorn) [inline styles/alpha.h:41]: Maybe add an example to make it clear how to use it?

Comment (fdarosa2663) [reply styles/alpha.h:41]: sure, added

**Comment by profezzorn** [comment]
I had to revert this change because some of the tests failed.
I think you need to add an #include "mix.h" to prevent that.

**Pull request merged**